src/helpers/lib/formatBN.js

```diff
@@ -4,18 +4,25 @@ export const tenPow = x => (
   (new BN(10)).pow(new BN(x))
 )
 
-export const formatBN = (amount, base, precision) => {
+export const formatBN = (amount, base, precision, fixed = false) => {
```

Comment:
```suggestion
export const formatBN = (amount, base, precision, fixed = true) => {
```

I think that keeping the default for fixed to true should make everything else that relies on `formatBN`(e.g. `formatPct`) working in a similar way as it was before.

Reply:
When setting `fixed` to `true` by default, the following tests break:

```ava
 2 tests failed

  examples › examples › 58 - Change required support to `@formatPct(support)`%

  /Users/schmidsi/Development/@aragon/radspec/test/examples/examples.js:365

   364:     )        
   365:     t.is(    
   366:       actual,

  Expected "Change required support to `@formatPct(support)`%" to evaluate to "Change required support to 50%", but evaluated to "Change required support to 50.00%"

  Difference:

  - 'Change required support to 50.00%'
  + 'Change required support to 50%'



  examples › examples › 59 - Change required support to `@formatPct(support, 10 ^ 18, 1)`%

  /Users/schmidsi/Development/@aragon/radspec/test/examples/examples.js:365

   364:     )        
   365:     t.is(    
   366:       actual,

  Expected "Change required support to `@formatPct(support, 10 ^ 18, 1)`%" to evaluate to "Change required support to 40.4%", but evaluated to "Change required support to ~40.4%"

  Difference:

  - 'Change required support to ~40.4%'
  + 'Change required support to 40.4%'
```

I had backwards compatibility in mind and tried to change as little of the tests as possible. But I don't know about the projects where this package is used. Does defaulting `fixed = true` break less tests in your other repos?

Reply:
It shouldn't break anything, no. I think it's fine going with false by default then